ceremonies/src/tests.rs

```diff
@@ -301,6 +301,9 @@ fn registering_participant_works() {
 		assert_eq!(EncointerCeremonies::bootstrapper_count((cid, cindex)), 0);
 		assert_ok!(register(alice.clone(), cid, None));
 
+		assert_eq!(last_event::<TestRuntime>(),
```

Comment:
@clangenb this assertion fails because no event was dispatched. I don't understand why that is the case (the real chain dispatches this event, I checked)